SingularityUI/app/components/common/JSONButton.jsx

```diff
@@ -27,10 +27,9 @@ export default class JSONButton extends Component {
 
     this.showJSON = this.showJSON.bind(this);
     this.hideJSON = this.hideJSON.bind(this);
-  }
 
-  componentDidMount() {
-    this.clipboard = new Clipboard('.copy-btn');
+    this.attachClipboard = this.attachClipboard.bind(this);
```

Comment:
@andyhuang91 , it is in the constructor, if you look a bit up in the diff, you'll see he's removing the `componentDidMount() {` line